war/src/main/less/base/style.less

```diff
@@ -1461,7 +1455,32 @@ DIV.yahooTree td {
 
 #jenkins .yuimenu .bd {
     border: 1px solid #ccc;
-    box-shadow: 0px 3px 10px #bbb;
+    box-shadow: var(--menu-box-shadow);
+}
+
+#jenkins.yui-skin-sam {
```

Comment:
Why did you need to create this? Does this fix something?

Reply:
it allows overriding the yui-skin in themes, I've matched the css below to what the yui skin provides and extended it with variables